.github/workflows/prettier-html.yml

```diff
@@ -0,0 +1,37 @@
+name: Pretty HTML
+
+on:
+  workflow_dispatch:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout gh-pages branch
+        uses: actions/checkout@v3
```

Comment:
Let's use the [latest version](https://github.com/actions/checkout?tab=readme-ov-file#usage) `v4` here.